common/transport/src/main/java/org/thingsboard/server/common/transport/adaptor/JsonConverter.java

```diff
@@ -54,6 +56,29 @@ public static TelemetryUploadRequest convertToTelemetry(JsonElement jsonObject,
         return request;
     }
 
+    // HMDC test telemetry depth.
+    public static TelemetryUploadRequestForDepth convertToTelemetryDepth(JsonElement jsonObject, int requestId) throws JsonSyntaxException {
+        BasicTelemetryUploadRequestForDepth request = new BasicTelemetryUploadRequestForDepth(requestId);
+        log.error("\n\n HMDC request class: " + request.getClass() + "\n\n");
+        log.error("\n\n HMDC request val " + request + "\n\n");
```

Comment:
These loggers can't be error, if you want make it debug or trace level like 
`log.debug("HMDC request class: {}", request.getClass());`